### ISSUE TITLE
Make all row settings mobile responsive

### ISSUE THREAD

**Issue by Bobolinos** [beaverbuilder/feature-requests#299]
There are still quite a few row settings that are not mobile responsive, can you please make these mobile responsive, for example the field vertical aligning. It's a bit annoying that this cannot be changed on mobile 🙂.

**Comment by mcneese** [comment]
It would be great to control the responsive behavior of shape dividers at different breakpoints.

**Comment by majamaki** [comment]
Agree. Having options to define different values for the Shape settings such as Shape Size and Transform would be very helpful for us as some shape settings that look great on desktop do not work so well in mobile or tablet screens, thus reducing the experience.
